src/components/SlateEditor/plugins/embed/SlateInputField.jsx

```diff
@@ -89,7 +89,7 @@ SlateInputField.propTypes = {
   onChange: PropTypes.func.isRequired,
   name: PropTypes.string.isRequired,
   label: PropTypes.string.isRequired,
-  required: PropTypes.bool.isRequired,
```

Comment:
Maybe set required as defaultProps false for clarity?

Reply:
Utført.